packages/core/src/components/dropdown-button/dropdown-button.template.ts

```diff
@@ -0,0 +1,38 @@
+import { DropdownButton } from "@dso-toolkit/sources";
+import { html } from "lit-html";
+
+export function dropdownButtonTemplate({ label, variant }: DropdownButton) {
```

Comment:
Deze xxxTemplate functions gebruiken we niet alleen om het component met testdata te tonen, maar ook als bouwstenen voor in andere componenten. Helemaal als er straks voorbeeldpagina's worden gemaakt is het fijn om (statically typed) een dropdown button te kunnen maken maar met andere content.

Je kan een extra property `children: TemplateFnReturnType` op `DropdownButton` model met typeargument `DropdownButton<TemplateFnReturnType>` plaatsen. In deze file geef je vervolgens `TemplateResult` van `lit-html` als type argument mee.

`children` kan je vervolgens met
```
html`
  <ul>....</ul>
`;
```

vullen.

Reply:
Aangepast